### DISCUSSION TITLE
In  admin of socket.io , it is possible that data is preserved in it ?

### DISCUSSION THREAD

**Discussion by Jagroop-RTE** [socketio/socket.io#4683] [General]
I have Integrated my socket.io with admin panel . But when I refresh the session expires and it clear all the stats in Event Section. Is it possible to maintain all the actions in Event Tab .

**Comment by darrachequesne** [comment]
Hi! This is not currently possible, but that should be doable. Could you please explain your use case?

**Comment by Jagroop-RTE** [comment]
Actually , our main reason to use Admin of Socket is that we can analyze the record of socket like how much percentage it fails, retries , takes time to load and successfully worked with their Details. But we observe that it will only show when we loggedIn and without refresh . After refresh all the data is disappeared.
